flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamConfigChainer.java

```diff
@@ -77,6 +86,7 @@ private void head(OperatorID headOperatorID) {
         headConfig.setChainIndex(chainIndex);
     }
 
+    @Deprecated
```

Comment:
nit: could you add a java-doc pointing to what should be used instead of those deprecate methods?